assets/base.css

```diff
@@ -1258,7 +1263,6 @@ details[open] > .share-button__fallback {
 .share-button__button,
 .share-button__fallback button {
   cursor: pointer;
-  background: rgb(var(--color-background));
   border: none;
```

Comment:
That might have been there specifically for Safari as from what I can see it adds a default bg colour when none is mentioned: ![](https://screenshot.click/21-08-ekqaq-4i4iu.png)

SO you can maybe re add it with a value of `transparent`

Reply:
Seems like on android too.